conclave-common/src/main/kotlin/com/r3/conclave/common/internal/attestation/AttestationJsonTypes.kt

```diff
@@ -423,10 +511,84 @@ enum class EnclaveTcbStatus {
 /**
  * @property isvsvn SGX Enclave’s ISV SVN.
  */
-data class EnclaveTcb @JsonCreator constructor(
-    @JsonProperty("isvsvn")
-    val isvsvn: Int
-)
+data class EnclaveTcb constructor(val isvsvn: Int) {
+    companion object {
+        fun fromJson(json: JsonNode): EnclaveTcb {
+            return EnclaveTcb(json.getInt("isvsvn"))
+        }
+    }
+}
+
+/**
+ * Utility functions for manual parsing of json primitives and producing sensible error messages.
+ */
+private fun JsonNode.checkFieldExists(fieldName: String) {
+    check(this.has(fieldName)) { "No such field $fieldName." }
```

Comment:
`require`. we want to it throw `IllegalArgumentException`. and likwise for the rest of the code